pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/index.md

```diff
@@ -8,6 +8,43 @@ excerpt: Workspace Catalog Applications
 <!-- TODO: settle down the link target to custom applications -->
 Catalog applications are any third-party or open source applications that appear in the Catalog. These applications can be provided by D2iQ for use in your environment.
 
+## Install the workspace catalog via the CLI
+
+Follow these steps to install the catalog from the CLI.
+
+1.  Set the `WORKSPACE_NAMESPACE` environment variable to the name of your workspace’s namespace:
+
+    ```bash
+    export WORKSPACE_NAMESPACE=<workspace namespace>
+    ```
+
+1.  Create the `GitRepository`:
+
+    ```sh
+    kubectl apply -f - <<EOF
+    apiVersion: source.toolkit.fluxcd.io/v1beta1
+    kind: GitRepository
+    metadata:
+      name: dkp-catalog-applications
+      namespace: ${WORKSPACE_NAMESPACE}
+      labels:
+        kommander.d2iq.io/gitapps-gitrepository-type: catalog
+        kommander.d2iq.io/gitrepository-type: catalog
```

Comment:
@takirala I added the recently required label, I think we still need both right?

Reply:
Yep! We need both for our dkp repo. For user-created custom apps, we just need the gitrepository-type label (gotta add that one to the docs for custom apps)

Reply:
perfect!